pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -1256,7 +1256,7 @@ func (r *Builder) PopulatorVolumes(vmRef ref.Ref, annotations map[string]string,
 						Annotations: annotations,
 					},
 					Spec: core.PersistentVolumeClaimSpec{
-						AccessModes:      []core.PersistentVolumeAccessMode{core.ReadWriteOnce},
+						AccessModes:      []core.PersistentVolumeAccessMode{core.ReadWriteMany},
```

Comment:
@mnecas I realize this is a bit unrelated. This was needed to be like that from the start I guess, to allow migration.

Reply:
@tatsumir following your request.

Reply:
Lets do it the proper way - a separate PR https://github.com/kubev2v/forklift/pull/1610